api/src/main/java/org/openmrs/api/ObsService.java

```diff
@@ -378,7 +378,9 @@ public Integer getObservationCount(List<Person> whom, List<Encounter> encounters
 	 * @should fill in complex data object for complex obs
 	 * @should return normal obs for non complex obs
 	 * @should not fail with null view
+	 * @deprecated as of 2.0, use {@link #getObs(Integer)} 
```

Comment:
The as of version should be the very first version which will be released with this deprecation. And in this case it is not 2.0 because this is already released without this deprecation. :)